Zend/zend_compile.h

```diff
@@ -245,6 +245,7 @@ typedef struct _zend_oparray_context {
 #define ZEND_ACC_INTERFACE               (1 <<  0) /*  X  |     |     |     */
 #define ZEND_ACC_TRAIT                   (1 <<  1) /*  X  |     |     |     */
 #define ZEND_ACC_ANON_CLASS              (1 <<  2) /*  X  |     |     |     */
+#define ZEND_ACC_ENUM                    (1 << 22) /*  X  |     |     |     */
```

Comment:
Should update the above comment to say 23 instead of 22: `/* Class Flags (unused: 23...)                            |     |     |     */`

Reply:
Lol, I manually looked through the flags and failed to see this comment. Could've saved me some time. Updated the comment.